templates/playlist/playlist.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<div class="container-fluid">
+    <div class="row">
+        <div class = "col-md-3">
+            <h2>{{playlist.title}}</h2>
+            <p>{{playlist.description}}</p>
+            <small>{{playlist.date_created}}</small><small>{{playlist.date_updated}}</small>
```

Comment:
don't fucking use <small> please

Never use a tag because of it's property, use it based on its identity